tools/pyclient/README.md

```diff
@@ -1,12 +1,17 @@
-# Installation
+The Molgenis EMX2 Pyclient is Python package developed to be used for data management on Molgenis EMX2 servers.
+
+## Installation
+The releases of the package are hosted at [PyPI](https://pypi.org/project/molgenis-emx2-pyclient/).

```

Comment:
Is the documentation built and stored anywhere?

Reply:
I am writing something up in docs/molgenis/use_usingapis.md, but it may require its own docs page there